apps/challenges/challenge_config_utils.py

```diff
@@ -189,15 +201,19 @@ def is_challenge_phase_split_mapping_valid(
     phase_id = phase_split["challenge_phase_id"]
     leaderboard_id = phase_split["leaderboard_id"]
     dataset_split_id = phase_split["dataset_split_id"]
-    if phase_id in phase_ids:
-        if leaderboard_id in leaderboard_ids:
-            if dataset_split_id in dataset_split_ids:
-                return True
-            else:
-                return False
-        else:
-            return False
-    return False
+    error_messages = []
+
+    if leaderboard_id not in leaderboard_ids:
+        error_messages.append("ERROR: Challenge phase split {} found leaderboard id {}. This relationship is not valid. Please check your configuration.".format(challenge_phase_split_index, leaderboard_id))
```

Comment:
Change message to `ERROR: Invalid leaderboard id {} found in challenge phase spit {}`